### PR TITLE
PAL performance

### PR DESCRIPTION
**Pull request by howard-stearns**
Picking up some low-hanging fruit effecting PAL performance:
- There was a bug that was adding a new audio timer each time it was opened.
- When there are more than 20 avatars:
  -- show just the avatar node, not the cloud, as that was busy visually, and added a  lot of model overlays.
  -- update the microphone only 1/3 as often.

### CODE REVIEW

**Comment by howard-stearns** [comment]
This doesn't do an exhaustive or qualitative review of where the time is spent. It's just catching some things that apparently matter.

**Comment by howard-stearns** [comment]
test plan:
- Open and close the pal several times, with at least 20 avatars present. (The current summon/crowd-agent scripts should be good.) The performance should be acceptable. 
- Make sure you still get the cloud overlay on avatars when there are less than 20.
- Optionally, compare against master, or try with 100 avatars in dev-chris.highfidelity.io (after checking with Sam that it is available).

File: scripts/system/pal.js [hunk 1]
```diff
@@ -521,7 +526,7 @@ function onClicked() {
         Controller.mouseMoveEvent.connect(handleMouseMoveEvent);
         triggerMapping.enable();
         triggerPressMapping.enable();
-        createAudioInterval();
+        audioTimer = createAudioInterval(AUDIO_LEVEL_UPDATE_INTERVAL_MS / (conserveResources ? 3 : 1));
```
Comment (davidkelly) [inline scripts/system/pal.js]: So, we have a smaller interval when conserveResources = true?  Seems like we'd be multiplying instead here.

Comment (howard-stearns) [inline scripts/system/pal.js]: doh!

**Comment by hifi-gustavo** [comment]
Build console output for commit: 23fc9b6fb33ac041c5a7912ae4f18c15d6d20fe5 available at: https://gist.github.com/df1556c2bb709d91fcd22bd6dc7f1b43<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9627-23fc9b6.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9627-23fc9b6.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2e88f1ab1fa6aecaf8b34e693a8f977e139bb5b2 available at: https://gist.github.com/df1556c2bb709d91fcd22bd6dc7f1b43<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9627-2e88f1a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9627-2e88f1a.dmg

**Pull request merged**